src/main/java/net/mcreator/ui/dialogs/TextureImportDialogs.java

```diff
@@ -119,6 +119,54 @@ public static void importArmor(final MCreator mcreator) {
 			}
 	}
 
+	public static void importHorseArmor(final MCreator mcreator) {

```

Comment:
Horse armor should be part of regular entity textures, so no need for a separate importer